common/src/main/java/com/wynntils/core/features/overlays/Overlay.java

```diff
@@ -47,6 +65,49 @@ public Overlay(
 
     public abstract void render(PoseStack poseStack, float partialTicks, Window window);
 
+    @Override
+    public final void updateConfigOption(ConfigHolder configHolder) {
+        // if user toggle was changed, enable/disable feature accordingly
+        if (configHolder.getFieldName().equals("userEnabled")) {
```

Comment:
Once again we're running into "enabled" as a special case. I'm getting more and more firm in my belief that the "enabled" status is not an aspect of the thing (feature, overlay) *itself* but rather on the owner (feature manager, or the feature owning the overlay (or possibly the overlay manager).

Reply:
Feel free to rework it :). Aside from the joke, yeah, perhaps that is better, once again I don't want to rewrite the whole Feature/Overlay impl this pr.